tests/Feature/SaleTest.php

```diff
@@ -0,0 +1,213 @@
+<?php
+
+namespace Tests\Feature;
+
+use App\Models\Sale;
+use App\Models\User;
+use Tests\TestCase;
+
+class SaleTest extends TestCase
+{
+    /**
+     *
```

Comment:
Ah one more, you forgot to add `uses DatabaseTransactions`, this test will actually add things to the database permanently